src/glibc/target/include/bits/setjmp.h

```diff
@@ -27,13 +27,7 @@
 
 #ifndef _ASM
 
-# if __WORDSIZE == 64
```

Comment:
Why is this change made?   Shouldn't you at least assert the __WORDSIZE is set as expected?

Reply:
Okay, this is all fine, but please explain this in comments in the code, so that this is clear to us ~3 months / years from now...   We will stumble across this code later and wonder why it was done this way in many cases.